database/migrations/2021_03_23_145012_add_number_length_setting.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Crater\Models\CompanySetting;
+use Crater\Models\User;
+
+class AddNumberLengthSetting extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        $user = User::where('role', 'super admin')->first();
+
+        if ($user) {
+            $invoice_number_length = CompanySetting::getSetting('invoice_number_length', $user->company_id);
+            if(empty($invoice_number_length)) {
+                CompanySetting::setSetting('invoice_number_length', '6', $user->company_id);
+            }
+
+            $estimate_number_length = Setting::getSetting('estimate_number_length', $user->company_id);
```

Comment:
@TheZoker Hey, I think you missed using CompanySetting on estimate_number_length.